superset-frontend/src/filters/components/Select/controlPanel.ts

```diff
@@ -93,7 +93,10 @@ const config: ControlPanelConfig = {
               resetConfig: true,
               affectsDataMask: true,
               renderTrigger: true,
-              description: t('Select first item by default'),
+              requiredFirst: true,
+              description: t(
+                'Select first item by default (requires repopulate Default Value if column field changed)',
```

Comment:
I think the text in parenthesis can be replaced with that above (we don't need to repopulate Default Value anymore).